index.html

```diff
@@ -72,8 +72,199 @@ <h1 class="intro">
         <img src="./Assets/Ellipse4.png" class="roll_ball" alt="roll_ball" />
         <img src="./Assets/Rectangle5.png" class="roll_bar" alt="roll_bar" />
         <img src="./Assets/Ellipse12.png" class="bowl" alt="bowl" />
-        <img src="./Assets/Ellipse13.png" class="hook" alt="hook"/>
+        <img src="./Assets/Ellipse13.png" class="hook" alt="hook" />
       </section>
     </div>
+    <section class="works-container">
+      <div class="work-item">
+        <img
+          class="work-card"
+          src="./Assets/restaurant.JPG"
+          alt="project card display"
+        />
+        <div class="description-container">
+          <h1>Ristorant Confusion</h1>
+          <p class="description">
+            A Restaurant publicity and order website<br />
+            We take inpiration from the World's best Cuisine to create a
+            lipsmacking and satisfying collection
+          </p>
+          <div class="pale-info-container">
+            <div class="pale-info">html</div>
+            <div class="pale-info">Css</div>
+            <div class="pale-info">JavaScript</div>
+          </div>
```

Comment:
-  [ ] Kindly fix the width of each added project to improve your UI design.
This is your design:
<img width="197" alt="image" src="https://user-images.githubusercontent.com/77190203/148648230-3b91ae07-da6d-4bb6-9510-ff964b2d113b.png">